.github/workflows/publish.yml

```
@@ -0,0 +1,66 @@
+name: Publish artifacts
+
+on:
+  release:
+    ### NOTE:
+    ###
+    ### There are multiple types of release trigger events.
+    ### Ref: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release
+    ###
+    ### Given that GitHub allows the creation of 'release'
+    ### and 'pre-release' publications to differentiate between
+    ### production and beta releases, the trigger type to use,
+    ### as it wraps them both, must be 'published'.
+    types: [published]
+
+jobs:
+
+  docker:
+    needs: []
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Set up GitHub Actions"
+      uses: actions/checkout@v2
+    - name: "Login to DockerHub"
+      uses: docker/login-action@v1
```

```suggestion
      if: github.event_name != 'pull_request'
      uses: docker/login-action@v1
```

Needed to keep this from failing on PRs c.f. https://github.com/scikit-hep/pyhf/pull/1463